app/Notifications/CheckoutConsumableNotification.php

```diff
@@ -80,7 +80,7 @@ public function via($notifiable)
             /**
              * Send an email if an email should be sent at checkin/checkout
              */
-            if ($this->item->checkin_email()) {
+            if ((method_exists($this->item, 'checkin_email')) && ($this->item->checkin_email())) {
```

Comment:
I think I would rather fix this in the Consumable-Model itself, by adding the checkin method there. Otherwise you'd get checkout notifications on assets, accessories and components, but on consumables they would be missing 🤔